### DISCUSSION TITLE
How I can found the code?

### DISCUSSION THREAD

**Discussion by peeter7** [nuxt/nuxt#26787] [Questions]
Hello everyone, 
My name is Peeter Jackson. I am visiting a website which is made by Nuxt and I also want to make the same of it but I have not found the code of the tool which is in the website. Please help me. The link to the website is heightcomparison.com. Also visit my own website [name generator](http://universalnamegenerator.com/) and give me a suggestion and earn reward.

**Comment by TheAlexLichter** [comment]
Most websites are not open source, so you probably can't find the code online. You could download the JS file and try to reverse engineer it but easier would be building it similarly - from scratch.

**Reply by peeter7** [reply]
How I can download the JS? I visited a website, the tool in it made by Nuxt and when I visited Nuxt here I saw your profile So I am messaging you. Please inform me I want to make this tool the link to the website  is heightcomparison.com
